game/hud/src/components/HUD/index.tsx

```diff
@@ -61,12 +61,20 @@ class HUD extends React.Component<HUDProps, HUDState> {
         }]
     }
   }
-  
+
   componentWillMount() {
+    window.removeEventListener("resize", this.handleResize);
```

Comment:
Its not a style thing, its an engineering thing, magic strings are an anti-pattern.